third_party/libadblockplus/grit/resources/elemhide.js

```diff
@@ -0,0 +1,26 @@
+function hideElement(element)
```

Comment:
copy-paste from core - to be fetched from core and reused (see task description above).